website/docs/r/trusted_signing_account.html.markdown

```diff
@@ -0,0 +1,75 @@
+---
+subcategory: "Trusted Signing"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_trusted_signing_account"
+description: |-
+  Manages a Trusted Signing Account.
+---
+
+# azurerm_trusted_signing_account
+
+Manages a Trusted Signing Account.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example"
+  location = "West Europe"
+}
+
+resource "azurerm_trusted_signing_account" "example" {
+  resource_group_name = "example"
+  location            = "West Europe"
+  sku {
+    name = "Basic"
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Trusted Signing. Changing this forces a new Trusted Signing Account to be created.
+
+* `location` - (Required) The Azure Region where the Trusted Signing should exist. Changing this forces a new Trusted Signing Account to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Trusted Signing should exist. Changing this forces a new Trusted Signing to be created.
+
+* `sku` - (Required) One `sku` blocks as defined below.
+
+---
+
+* `tags` - (Optional) A mapping of tags which should be assigned to the Trusted Signing.
+
+---
+
+A `sku` block supports the following:
+
+* `name` - (Required) The sku name of this Trusted Signing Account.
```

Comment:
As this is the only attribute nested, can we flatten it one level upwards and rename it to `sku_name`?

Reply:
updated